src/CompositeFixtureLoader.php

```diff
@@ -0,0 +1,32 @@
+<?php
+namespace Lakion\ApiTestCase;
+
+use Nelmio\Alice\Fixtures;
+
+class CompositeFixtureLoader
```

Comment:
You are right. But I want to implement composite  pattern to accessing group of some objects which have same interface over single class for reducing touched section about loading fixtures. Did you have any suggestion in this aproach ? Maybe I can move these classes (CompositeFixtureLoader and CompositeObjectPurger) other sub namespace if you want.